doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/configuring-security-aspects.md

```diff
@@ -1 +1,522 @@
+      },
+      "iot_sensors": [],
+      "response_type": "code",
+      "client_type": "confidential",
+      "grant_type": "authorization_code",
+      "id": "36b34b9f2a3048c58c0a2763b5a3df0c",
+      "description": "Fiware TourGuide Application"
+    }
+  ]
+}
+```
+
+Or generate your own as explained [here](http://docs.keyrock.apiary.io/#reference/keystone-extensions/consumers/create-a-consumer)
+
+#### Roles
+
+The following list shows the roles generated:
+
+| Role name           | Granted to user                         |
+|---------------------|-----------------------------------------|
+| Provider            | pepproxy@test.com                       |
+| End user            | All                                     |
+| Franchise Manager   | user0@test.com     (Franchise1)         |
+| Franchise Manager   | user1@test.com     (Franchise2)         |
+| Franchise Manager   | user2@test.com     (Franchise3)         |
+| Franchise Manager   | user3@test.com     (Franchise4)         |
+| Global Manager      | user0@test.com                          |
+
+
+You can retrieve them by executing the following query:
+
+```
+curl -X GET -H "Content-Type: application/json" -H "X-auth-token: ADMIN" "http://keyrock:5000/v3/OS-ROLES/roles/"
+```
+
+Showing the following output:
+
+```
+{
+  "links": {
+    "self": "http://keyrock:5000/v3/OS-ROLES/roles",
+    "previous": null,
+    "next": null
+  },
+  "roles": [
+    {
+      "is_internal": true,
+      "application_id": "idm_admin_app",
+      "id": "provider",
+      "links": {
+        "self": "http://keyrock:5000/v3/OS-ROLES/roles/provider"
+      },
+      "name": "Provider"
+    },
+    {
+      "is_internal": true,
+      "application_id": "idm_admin_app",
+      "id": "purchaser",
+      "links": {
+        "self": "http://keyrock:5000/v3/OS-ROLES/roles/purchaser"
+      },
+      "name": "Purchaser"
+    },
+    {
+      "is_internal": false,
+      "application_id": "36b34b9f2a3048c58c0a2763b5a3df0c",
+      "id": "17d245ab695847f1800df8f85b360df9",
+      "links": {
+        "self": "http://keyrock:5000/v3/OS-ROLES/roles/17d245ab695847f1800df8f85b360df9"
+      },
+      "name": "End user"
+    },
+    {
+      "is_internal": false,
+      "application_id": "36b34b9f2a3048c58c0a2763b5a3df0c",
+      "id": "a5b6a9daa0594f8d818e3a83da5a498e",
+      "links": {
+        "self": "http://keyrock:5000/v3/OS-ROLES/roles/a5b6a9daa0594f8d818e3a83da5a498e"
+      },
+      "name": "Franchise manager"
+    },
+    {
+      "is_internal": false,
+      "application_id": "36b34b9f2a3048c58c0a2763b5a3df0c",
+      "id": "0efd09a12f074f63abe53ee943cfa6f5",
+      "links": {
+        "self": "http://keyrock:5000/v3/OS-ROLES/roles/0efd09a12f074f63abe53ee943cfa6f5"
+      },
+      "name": "Global manager"
+    }
+  ]
+}
+
+```
+Or generate some as explained [here](http://docs.keyrock.apiary.io/#reference/keystone-extensions/roles/create-a-role).
+
+#### Permissions
+
+Permissions can be listed by doing:
```

Comment:
Actually the permissions are the same. Authzforce policies are based in these permissions.